sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileClient.java

```diff
@@ -496,6 +499,57 @@ public FileReadResponse readWithResponse(OutputStream stream, FileRange range, D
         }, logger);
     }
 
+    /**
+     * Opens a file input stream to download the file.
+     * <p>
+     *
+     * @return An <code>InputStream</code> object that represents the stream to use for reading from the file.
```

Comment:
If you want to use it there is also a Javadoc keyword for code, `{@code }`

```suggestion
     * @return An {@code InputStream} object that represents the stream to use for reading from the file.
```

But `{@link }` might be a better use here as it'll create a hyperlink to the `InputStream` javadocs.

```suggestion
     * @return An {@link InputStream} object that represents the stream to use for reading from the file.
```